cmd/dlt/cluster/cmd.go

```diff
@@ -85,6 +85,11 @@ func run(cmd *cobra.Command, _ []string) {
 
 	cluster := r.FetchCluster()
 
+	if args.bestEffort {
+		r.Reporter.Warnf("Deleting a cluster with 'best effort' means that certain resources may be left behind " +
```

Comment:
@oriAdler Wondering if we should include the clusterKey and AWS account details in this message so something like:

```
r.Reporter.Warnf("Deleting cluster '%s' with 'best effort' means that certain resources may be left behind in AWS account '%s'. These resources will need to be deleted manually", r.GetClusterKey(),r.Creator.AccountID)
 ```

Reply:
Good point, added, thanks.